feature/sessions/src/commonMain/kotlin/io/github/droidkaigi/confsched/sessions/components/TimetableItemDetailContent.kt

```diff
@@ -74,18 +75,22 @@ private fun DescriptionSection(
     var isOverFlow by remember { mutableStateOf(false) }
 
     Column(modifier = Modifier.padding(8.dp)) {
-        SelectionContainer {
-            ClickableLinkText(
-                content = description,
-                regex = "(https)(://[\\w/:%#$&?()~.=+\\-]+)".toRegex(),
-                onLinkClick = onLinkClick,
-                style = MaterialTheme.typography.bodyLarge,
-                maxLines = if (isExpand) Int.MAX_VALUE else 7,
-                overflow = if (isExpand) TextOverflow.Clip else TextOverflow.Ellipsis,
-                onOverflow = {
-                    isOverFlow = it
-                },
-            )
+        provideSelectionContainerCustomContextMenu(
+            onWebSearchClick = { onLinkClick(it) },
```

Comment:
Since the link wasn't clicked, calling this method feels a bit off, but it risks violating the concerns likely raised in the comments below, so I've reluctantly implemented it this way.
https://github.com/DroidKaigi/conference-app-2025/pull/554#discussion_r2325326310